models/topics.model.js

```diff
@@ -8,3 +8,5 @@ exports.fetchAllTopics = () => {
       throw error;
     });
 };
+
```

Comment:
you don't need a catch block in your model, if the model function invocation in the controller fails then the catch block in the controller will catch that and send that on for you anyway. Only put custom errors in your model where needed